data/ai/micro_ais/scenarios/dungeon_mai_comparisons.cfg

```diff
@@ -25,7 +25,44 @@
     {DEEP_UNDERGROUND}
     {DEFAULT_MUSIC_PLAYLIST}
 
-    map_file=ai/micro_ais/maps/dungeon_mai_comparisons.map
+    # map_file=ai/micro_ais/maps/dungeon_mai_comparisons.map
```

Comment:
I'm guessing this would work if you add `data/` in front.

Reply:
I rectified this when I found it was due to me not pushing the changes from my fork to the PR and then pulling to my local build